migrations/versions/0183_alter_primary_key.py

```diff
@@ -43,8 +43,8 @@ def upgrade():
         to_char(datum, 'iyyy/IW') AS year_calendar_week,
         (SELECT CASE WHEN (extract(month from datum) <= 3) THEN (extract(year FROM datum) -1)
         ELSE (extract(year FROM datum)) end),
-        (datum + TIME '00:00:00') at Time zone 'Europe/London' at TIME zone 'utc' as utc_daytime_start,	-- convert bst time to utc time
-        (datum + TIME '24:00:00') at Time zone 'Europe/London' at TIME zone 'utc' as utc_daytime_end
+        (datum + TIME '00:00:00') at Time zone 'America/New_York' at TIME zone 'utc' as utc_daytime_start,	-- convert bst time to utc time
+        (datum + TIME '24:00:00') at Time zone 'America/New_York' at TIME zone 'utc' as utc_daytime_end
```

Comment:
I don't think we want to change this migration?

Reply:
Hmmm, I think this is correct since `bst_date` in the model is actually `est_date` (the confusion you mentioned above). Please let me know if my logic is off, though.

But either way, this table is deleted in `0307`, so I'm happy with whatever.

Reply:
Yeah, our migrations have been edited several times since the fork, but I agree that it's best if we avoid that.